ydb/library/yql/minikql/comp_nodes/ut/mkql_block_map_join_ut.cpp

```diff
@@ -628,23 +643,24 @@ Y_UNIT_TEST_SUITE(TMiniKQLBlockMapJoinBasicTest) {
         TSetup<false> setup;
         TProgramBuilder& pgmBuilder = *setup.PgmBuilder;
         // 1. Make input for the "left" stream.
-        TVector<ui64> keyInit(testSize);
+        TVector<std::optional<ui64>> keyInit(testSize);
```

Comment:
Please, add the totally new suite for optional keys with tests for each join type.

Reply:
For the history: [fixed](https://github.com/ydb-platform/ydb/pull/9847/commits/acc250bc7648b302ef788990e0b76367e1351d91).